src/python/lambdas.Dockerfile

```diff
@@ -0,0 +1,50 @@
+# This defines a base image for running Python applications in a container
+# meant to simulate AWS Lambda locally.
+#
+# This assumes the applications have previously been built and is only for
+# creating the deployment containers.
+#
+# If we are to continue using AWS Lambda we should consider switching from
+# deploying zips to deploying container images built from
+# public.ecr.aws/lambda/python:3.7.  See https://gallery.ecr.aws/lambda/python
+FROM python:3.7-slim-buster AS grapl-python-lambda-runner-base
+
+RUN --mount=type=cache,target=/var/lib/apt/lists \
+    apt-get update && apt-get install -y --no-install-recommends \
+        unzip
+
+RUN adduser \
+    --disabled-password \
+    --gecos '' \
+    --home /home/grapl \
+    --shell /bin/bash \
+    grapl
+
+USER grapl
+ENV USER=grapl
+WORKDIR /home/grapl
+RUN mkdir -p /home/grapl/bin
+RUN mkdir -p /home/grapl/.aws
+RUN echo "[default]\naws_access_key_id=fake\naws_secret_access_key=fake" > /home/grapl/.aws/credentials
```

Comment:
🤔 I know this wasn't introduced in this PR, but why are these being baked into the image, rather than being passed in as environment variables like we do with every other container in our test environments?

(cc: @jgrillo-grapl)

Reply:
I believe in my testing I found that if the file wasn't present on disk then the boto3 library threw an error, but it was a long time ago so I don't really remember.